packages/next-plugin-eslint/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@next/eslint-plugin-next",
+  "version": "0.0.0",
+  "description": "ESLint plugin for NextJS.",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "url": "zeit/next.js",
+    "directory": "packages/next-plugin-eslint"
+  },
+  "scripts": {
+    "test": "mocha tests --recursive"
```

Comment:
Ideally we'll want to use jest or our global test suite instead of having it inside the package